app/Http/Controllers/Front/ClientController.php

```diff
@@ -321,6 +318,19 @@ class='btn  btn-primary btn-xs' style='margin-right:5px;'>
         }
     }
 
+    public function getClientPanelOrdersData()
+    {
+        return Order::leftJoin('products','products.id','=','orders.product')
+            ->leftJoin('subscriptions','orders.id','=','subscriptions.order_id')
+            ->leftJoin('invoices','orders.invoice_id','invoices.id')
+            ->select('products.name as product_name','products.github_owner','products.github_repository','products.type','products.id as product_id','orders.id','orders.number','orders.client','subscriptions.id as sub_id','subscriptions.version','subscriptions.update_ends_at','products.name','orders.client','invoices.id as invoice_id','invoices.number as invoice_number')
+            ->where('orders.client',\Auth::user()->id)
+            ->get()->map(function($element){
```

Comment:
Do you need all the products in one shot? no pagination or limit?

Reply:
This is for client invoice which won't be much. Max 5 to 15 invoices. We do not require paginate for that.